source/features/open-issue-to-latest-comment.tsx

```diff
@@ -9,6 +11,14 @@ function init(): void {
 	}
 }
 
+function initDashboard(): void {
+	for (const icon of select.all('.js-recent-activity-container :not(a) > .octicon-comment')) {
+		const url = new URL(select<HTMLAnchorElement>('a', icon.closest('li')!)!.href);
```

Comment:
Don't use `select` and `closest` in the same line, it's unreadable. Also tag-only selectors automatically set the type

```suggestion
		const url = new URL(icon.closest('li')!.querySelector('a')!.href);
```

Reply:
> Also tag-only selectors automatically set the type

Good to know thanks